pymc3/distributions/continuous.py

```diff
@@ -1926,11 +1922,7 @@ def logp(self, value):
         -------
         TensorVariable
         """
-        nu = self.nu
-        mu = self.mu
-        lam = self.lam
-        sigma = self.sigma
-
+        lam, sigma = get_tau_sigma(tau=lam, sigma=None)
```

Comment:
We don't need to get the `sigma` again just to check if it's positive in the bound. The `lam` should have the same sign as `sigma`.